manifests/charts/gateways/istio-ingress/templates/service.yaml

```diff
@@ -10,6 +10,9 @@ metadata:
     {{ $key }}: {{ $val | quote }}
     {{- end }}
   labels:
+{{- if .Values.global.network }}
+    topology.istio.io/network: {{ .Values.global.network }}
```

Comment:
What do you mean here? I understand this is a pod label

Reply:
This is temporary just to get things working. We need to tie the gateway Service to its network. Right now it's bound to cluster. Another option would involve inferring the network by looking at Pods selected by the service.